src/createMap.ts

```diff
@@ -1,74 +1,84 @@
-import React, { useEffect, useRef, useState } from 'react';
 import mapboxgl from 'mapbox-gl';
-import 'mapbox-gl/dist/mapbox-gl.css';
-import Cat from '../Interfaces/Cat';
-import Coordinates from '../Types/Coordinates';
+import Cat from './Interfaces/Cat';
 
-export default function Map({ cats }: { cats: Cat[] }) {
-	const mapContainer = useRef<HTMLDivElement | null>(null); //  Lets you create a reference to a DOM element that stays between renders without triggering a re-render
-	const map = useRef<mapboxgl.Map | null>(null);
+export default function map(
```

Comment:
Does this function return a map or does it add things to an existing map? Probably should be called something like annotateMap